lms/templates/courseware/proctored-exam-status.underscore

```diff
@@ -19,9 +19,14 @@
             <% } %>
         </span>
         <span class="sr timer-announce" aria-live="assertive"></span>
-        <span id="time_remaining_id">
-            <b>
-            </b>
+        <span class="exam-timer-clock">
+            <span id="time_remaining_id">
+                <b>
+                </b>
+            </span>
+            <button id="toggle_timer" title="<%- gettext("Hide Timer") %>" aria-label="<%- gettext("Hide Timer") %>">
```

Comment:
@cptvitamin Is either of those preferable? Or is there something other than a `<button>` I could use so that the `title` wouldn't be read by a screen reader?